--- glog-0.3.3/src/demangle_unittest.cc.orig	2012-01-11 10:14:17.000000000 +0100
+++ glog-0.3.3/src/demangle_unittest.cc	2015-02-08 21:29:01.660388457 +0100
@@ -46,6 +46,7 @@ GLOG_DEFINE_bool(demangle_filter, false,
 
 using namespace std;
 using namespace GOOGLE_NAMESPACE;
+using namespace gflags;
 
 // A wrapper function for Demangle() to make the unit test simple.
 static const char *DemangleIt(const char * const mangled) {
--- glog-0.3.3/src/logging_unittest.cc.orig	2013-01-31 05:39:40.000000000 +0100
+++ glog-0.3.3/src/logging_unittest.cc	2015-02-08 21:24:53.070398888 +0100
@@ -79,6 +79,7 @@ using GOOGLE_NAMESPACE::glog_testing::Sc
 
 using namespace std;
 using namespace GOOGLE_NAMESPACE;
+using namespace gflags;
 
 // Some non-advertised functions that we want to test or use.
 _START_GOOGLE_NAMESPACE_
--- glog-0.3.3/src/signalhandler_unittest.cc.orig	2012-01-11 10:14:17.000000000 +0100
+++ glog-0.3.3/src/signalhandler_unittest.cc	2015-02-08 21:30:07.120385709 +0100
@@ -42,6 +42,7 @@
 #include "glog/logging.h"
 
 using namespace GOOGLE_NAMESPACE;
+using namespace gflags;
 
 void* DieInThread(void*) {
   // We assume pthread_t is an integral number or a pointer, rather
